sdv/metadata/metadata_upgrader.py

```diff
@@ -40,7 +40,7 @@ def _upgrade_columns_and_keys(old_metadata):
                 if regex_format:
                     column_meta['regex_format'] = regex_format
 
-            if field != primary_key:
```

Comment:
Is this where foreign keys were being added as alternate keys? If so nice catch

Reply:
yes